plugins/MachineSettingsAction/MachineSettingsAction.qml

```diff
@@ -164,7 +164,7 @@ Cura.MachineAction
                                 id: gcodeFlavorComboBox
                                 sourceComponent: comboBoxWithOptions
                                 property string settingKey: "machine_gcode_flavor"
-                                property string label: catalog.i18nc("@label", "GCode flavor")
+                                property string label: catalog.i18nc("@label", "Gcode flavor")
```

Comment:
> noticed fdmprinter.def.json (where most of the tooltips come from) is (mostly) standardised in "Gcode".

Probably due to preferences of the guy who writes these new settings ;) And lack of scrutiny with the guy who has to review that.

I believe I've always used "g-code" and capitalised that at the start of the sentence.